### ISSUE TITLE
"ember install ember-cli-foundation-6-sass" hangs PowerShell

### ISSUE THREAD

**Issue by badraufran** [acoustep/ember-cli-foundation-6-sass#19]
I'm using Windows PowerShell with administrative rights on Windows 8.1.

The command-line hangs at "Installing packages for tooling via npm"  after entering "ember install ember-cli-foundation-6-sass".

This is the output after entering the command:
`ember install ember-cli-foundation-6-sass`
```
Installed packages for tooling via npm.
installing ember-cli-foundation-6-sass
  install bower package foundation-sites
  cached https://github.com/zurb/foundation-sites.git#6.1.2
Installed browser packages via Bower.
  identical app\styles\app.scss
  install packages ember-cli-sass, broccoli-clean-css
  Installing packages for tooling via npm
```

All of the node modules and bower components are installed.

After I `ctrl+c` to terminate the command, it appears the ember-cli-foundation-6-sass module and foundation-sites component has been successfully installed with all of the files.

Is this an issue? I'm not sure if the hanging means the module isn't installed correctly. I haven't tried this using the command prompt.

**Comment by acoustep** [comment]
Unfortunately I don't have a Windows 8.1 machine to test this on. I'll try it on my Windows 10 machine tonight though.

What version of Node are you running?

**Comment by badraufran** [comment]
node: 4.4.1
ember-cli: 2.4.3

**Comment by acoustep** [comment]
Hi @badraufran 

I have managed to replicate your issue on Windows 10.  I believe this could be related to either ember-cli-sass or broccoli-clean-css.

I tried installing Node 5.10.1 with the latest ember-cli and it worked fine.

I realise this isn't ideal, however my best advice is to use something like [nvm-windows](https://github.com/coreybutler/nvm-windows) and either use the latest version of Node or try 4.4.3 which is the latest LTS.
